app/models/licence.model.js

```diff
@@ -9,6 +9,8 @@ const { Model } = require('objection')
 
 const BaseModel = require('./base.model.js')
 
+const licenceRef = 'licences.licenceRef'
```

Comment:
I know SonarCloud made you do this. But there are occasions where we are happy to tell SonarCloud to go 'do one' and this is one of them.

If we do it for one we should do it for all, and we think it just makes the models more complex and less maintainable.

Would you mind reverting this change? Ping me and I can silence SonarCloud when it moans again.